packages/npm-dependencies-validation/package.json

```diff
@@ -24,5 +24,8 @@
     "clean": "rimraf ./dist",
     "test": "mocha",
     "ci": "npm-run-all clean compile coverage"
+  },
+  "dependencies": {
+    "@types/vscode": "^1.50.0"
```

Comment:
npm-dep-validation should not depend on vscode (even types).
The logging/outputChannel dep should be in a more generic fashion
e.g a generic logger interface that the consumer (our vscode ext) implements.

Reply:
Done